scripts/translate/de_DE.po

```diff
@@ -2536,7 +2536,7 @@ msgstr "~! EILE"
 #: orc/buttons.lua:230
 #, fuzzy
 msgid "~!RAISE DEAD"
-msgstr "~! TOTE AUFERWECKEN"
+msgstr "~! TOTE ERWECKEN"
```

Comment:
```suggestion
msgstr "TOTE ERWECKEN (~!R)"
```

Reply:
If I am correct, it is used to show shortcut/hotkey. And there are not adapted to translation...
So mismatch might be seen as "error".

Reply:
All right. Based on the logic, that `~!` is a prefix marker for hotkeys, I fixed some more hotkeys. :)